operator/src/main/java/org/bf2/operator/managers/IngressControllerManager.java

```diff
@@ -492,18 +490,6 @@ private void buildIngressController(String name, String domain,
 
         Optional<IngressController> optionalExisting = Optional.ofNullable(existing);
         IngressControllerBuilder builder = optionalExisting.map(IngressControllerBuilder::new).orElseGet(IngressControllerBuilder::new);
-        Integer existingReplicas = optionalExisting.map(IngressController::getSpec).map(IngressControllerSpec::getReplicas).orElse(null);
```

Comment:
Note to others - we decided that this can be removed for now as we'll likely run with a single replica most of the time.  Until we are dealing with larger instance counts, or we need to support a higher amount of peak demand - we'll run with 1.  So it's highly unlikely that we need to dampen the scale down.